### ISSUE TITLE
Error pops up every time I create a new game

### ISSUE THREAD

**Issue by Darkman2371** [textadventures/quest#1066]
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at TextAdventures.Quest.EditorControls.WFEditorTree.SetMenuEnabled(String key, Boolean enabled)
   at TextAdventures.Quest.Editor.UpdateClipboardButtons()
   at TextAdventures.Quest.Editor.Redisplay()
   at TextAdventures.Quest.Main.ctlPlayer_Quit()
   at TextAdventures.Quest.Player._Lambda$__182-0()

**Comment by ThePix** [comment]
Don't think I've seen that before, and the error message is none too helpful.

What version of Windows and Quest are you using?
Can you play downloaded games?
Have you tried re-installing Quest?
Has Quest worked previously? If so, what version?

**Comment by Darkman2371** [comment]
Hi

I’m on Windows 10
Quest version 5.8.0

The error occurs when creating a new game and when the UI comes up. If you ignore the error and load a previous game the error also pops up. 

The previous working version I used was 5.7.1

However I have since tried uninstalling and wiping the installation directory and re-installing. Which has now solved the issue. 

Thanks for contacting me. 

Regards
Mark
